### ISSUE TITLE
Replies take too long to load

### ISSUE THREAD

**Issue by mplorentz** [planetary-social/planetary-ios#572]

**Comment by martindsq** [comment]
We should show a loading indicator like how we do the follow messages.

**Comment by danlatorre** [comment]
Is this fixed now by moving to Swift?

**Comment by mplorentz** [comment]
@danlatorre no, we haven't rewritten the replies view in SwiftUI yet (#1062). The UI may feel snappier after we do if we do incremental loading of replies, so maybe we should re-evaluate the priority of this ticket after that.
